### PR TITLE
New instructions

### PR DESCRIPTION
**Pull request by feliam**
New istanbul instructions.
CHAINID
SELFBALANCE
EXTCODEHASH

### CODE REVIEW

File: manticore/platforms/evm.py [hunk 1]
```diff
@@ -1832,6 +1841,10 @@ def GASLIMIT(self):
         """Get the block's gas limit"""
         return self.world.block_gaslimit()
 
+    def CHAINID(self):
+        """Get current chainid."""
+        return 0
```
Comment (montyly) [inline manticore/platforms/evm.py]: Should we return `1` by default? It's the value of mainet: https://chainid.network/

Comment (feliam) [reply manticore/platforms/evm.py]: fixed

**Comment by montyly** [comment]
Looks good to me. 

I think `setup.py` needs to update `pyevmasm` to `0.2.2`, so `istanbul` can be used (currently the CI is crashing because of that)

**Comment by ehennenfent** [comment]
Will this work if we bump the pyevmasm version to 0.2.3 (now on PyPI)?

File: setup.py [hunk 2]
```diff
@@ -50,15 +50,15 @@ def rtd_dependent_deps():
     python_requires=">=3.6",
     install_requires=[
         "pyyaml",
-        # evm dependencies
+        "wrapt",
```
Comment (ehennenfent) [inline setup.py:53]: We removed wrapt in #1672. Merge conflict?

**Pull request merged**